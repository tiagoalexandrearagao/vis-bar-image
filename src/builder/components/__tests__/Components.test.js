Object.defineProperty(exports, "__esModule", { value: true });
var Components = require("..");
var Configs_1 = require("./Configs");
var ListConfigs_1 = require("./Configs/ListConfigs");
describe("Test Table", function () {
  Configs_1.TABLE_CONFIGS.forEach(function (config, index) {
    it("Table Config ".concat(index), function () {
      var TABLE = Components.getTable(config);
      expect(TABLE.replace(/\n*\t*\r*\s+/g, " ")).toMatchSnapshot();
    });
  });
});
describe("Test Card", function () {
  Configs_1.CARD_CONFIGS.forEach(function (config, index) {
    it("Card Config ".concat(index), function () {
      var CARD = Components.getCard(config);
      expect(CARD.replace(/\n*\t*\r*\s+/g, " ")).toMatchSnapshot();
    });
  });
});
describe("Test Container", function () {
  Configs_1.CONATINER_CONFIGS.forEach(function (config, index) {
    it("Card Config ".concat(index), function () {
      var CONTAINER = Components.getContainer(config);
      expect(CONTAINER.replace(/\n*\t*\r*\s+/g, " ")).toMatchSnapshot();
    });
  });
});
describe("Test Chip", function () {
  Configs_1.CHIP_CONFIGS.forEach(function (config, index) {
    it("Card Config ".concat(index), function () {
      var CHIP = Components.getChip(config);
      expect(CHIP.replace(/\n*\t*\r*\s+/g, " ")).toMatchSnapshot();
    });
  });
});
describe("Test List", function () {
  ListConfigs_1.LIST_CONFIGS.forEach(function (config, index) {
    it("List Config ".concat(index), function () {
      var LIST = Components.getList(config);
      expect(LIST.replace(/\n*\t*\r*\s+/g, " ")).toMatchSnapshot();
    });
  });
});
describe("Test Progress Bar", function () {
  Configs_1.PROGRESSBAR_CONFIGS.forEach(function (config, index) {
    it("Progress Bar Config ".concat(index), function () {
      var BAR = Components.getProgressBar(config);
      expect(BAR.replace(/\n*\t*\r*\s+/g, " ")).toMatchSnapshot();
    });
  });
});
describe("Test Text", function () {
  Configs_1.TEXT_CONFIGS.forEach(function (config, index) {
    it("Text Config ".concat(index), function () {
      var TEXT = Components.getText(config);
      expect(TEXT.replace(/\n*\t*\r*\s+/g, " ")).toMatchSnapshot();
    });
  });
});
