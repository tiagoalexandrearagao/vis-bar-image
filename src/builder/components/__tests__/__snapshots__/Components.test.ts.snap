// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Card Card Config 0 1`] = `" <div class='paper card ' style=' width: fit-content;height: auto;margin: 4px 4px; ; ; '> <div class='cardHead ' style= ' background: rgba(0,0,0,.03); color: rgba(0,0,0,.54); text-align: left; ; '> Heading </div> <div class='cardContents' style= 'text-align:left; ; '> This is the contents of the card... </div> </div> "`;

exports[`Test Card Card Config 1 1`] = `" <div class='paper card ' style=' width: fit-content;height: auto;margin: 4px 4px; ; ; '> <div class='cardHead shadow' style= ' background: rgba(0,0,0,.03); color: rgba(0,0,0,.54); text-align: left; ; '> Heading </div> <div class='cardContents' style= 'text-align:left; ; '> This is the contents of the card... </div> </div> "`;

exports[`Test Card Card Config 2 1`] = `" <div class='paper card ' style=' width: fit-content;height: auto;margin: 4px 4px; ; ; '> <div class='cardHead shadow' style= ' background: #2E7D32; color: #fff; text-align: center; ; '> Heading </div> <div class='cardContents' style= 'text-align: center; ; '> This is the contents of the card... </div> </div> "`;

exports[`Test Card Card Config 3 1`] = `" <div class='paper card shadow' style=' width: fit-content;height: auto;margin: 4px 4px; ; ; '> <div class='cardHead shadow' style= ' background: rgba(0,0,0,.03); color: rgba(0,0,0,.54); text-align: center; ; '> Heading </div> <div class='cardContents' style= 'text-align: center; ; '> This is the contents of the card... </div> </div> "`;

exports[`Test Card Card Config 4 1`] = `" <div class='paper card ' style=' width: fit-content;height: auto;margin: 4px 4px; ; border: 1px solid #1565C0; '> <div class='cardHead ' style= ' background: #AD1457; color: #fff; text-align: center; ; '> Heading </div> <div class='cardContents' style= 'text-align: center; ; '> This is the contents of the card... </div> </div> "`;

exports[`Test Chip Card Config 0 1`] = `"<div class='chip'><span class='chipText'>text inside a chip</span></div>"`;

exports[`Test Container Card Config 0 1`] = `" <div class='container' style=' justify-content:center; background-color: #B2DFDB; width: 300px; height: 300px; ; '> <div style='margin-top: auto;'> text inside a container</div> </div> "`;

exports[`Test Container Card Config 1 1`] = `" <div class='container' style=' justify-content:center; background-color: #F5F5F5; width: 300px; height: 300px; ; '> <div style='margin-bottom: auto;margin-top: auto;'> text inside a container</div> </div> "`;

exports[`Test Container Card Config 2 1`] = `" <div class='container' style=' justify-content:flex-start; background-color: #FFE0B2; width: 300px; height: 300px; ; '> <div style='margin-top: auto;'> text inside a container</div> </div> "`;

exports[`Test Container Card Config 3 1`] = `" <div class='container' style=' justify-content:flex-start; background-color: #fff; width: 300px; height: 300px; ; '> <div style='margin-bottom: auto;margin-top: auto;'> text inside a container</div> </div> "`;

exports[`Test Container Card Config 4 1`] = `" <div class='container' style=' justify-content:flex-start; background-color: #F8BBD0; width: 300px; height: 300px; ; '> <div style='margin-bottom: auto;'> text inside a container</div> </div> "`;

exports[`Test Container Card Config 5 1`] = `" <div class='container' style=' justify-content:flex-end; background-color: #C8E6C9; width: 300px; height: 300px; ; '> <div style='margin-top: auto;'> text inside a container</div> </div> "`;

exports[`Test Container Card Config 6 1`] = `" <div class='container' style=' justify-content:flex-end; background-color: #fff; width: 300px; height: 300px; ; '> <div style='margin-bottom: auto;margin-top: auto;'> text inside a container</div> </div> "`;

exports[`Test Container Card Config 7 1`] = `" <div class='container' style=' justify-content:flex-end; background-color: #C5CAE9; width: 300px; height: 300px; ; '> <div style='margin-bottom: auto;'> text inside a container</div> </div> "`;

exports[`Test Container Card Config 8 1`] = `" <div class='container' style=' justify-content:center; background-color: #F8BBD0; width: 300px; height: 300px; ; '> <div style='margin-bottom: auto;'> text inside a container</div> </div> "`;

exports[`Test List List Config 0 1`] = `" <div class='listCard ' style=' width: fit-content;height: auto;margin: 4px 4px;; background-color: #fff; ; undefined; ' > <div class = 'listTitle' style=' ; undefined;' > List Title </div> <div class='listItems' style=' ; undefined; ' > item 1 </div><div class='listItems' style=' ; undefined; ' > items 2 </div><div class='listItems' style=' ; undefined; ' > item 3 </div></div>"`;

exports[`Test List List Config 1 1`] = `" <div class='listCard ' style=' width: fit-content;height: auto;margin: 4px 4px;; background-color: #B2DFDB; border-left: 2px solid #1565C0; undefined; ' > <div class = 'listTitle' style=' ; undefined;' > List Title </div> <div class='listItems' style=' ; undefined; ' > item 1 </div><div class='listItems' style=' ; undefined; ' > items 2 </div><div class='listItems' style=' ; undefined; ' > item 3 </div></div>"`;

exports[`Test List List Config 2 1`] = `" <div class='listCard shadow' style=' width: fit-content;height: auto;margin: 4px 4px;; background-color: #C5CAE9; border-left: 2px solid #1565C0; undefined; ' > <div class = 'listTitle' style=' ; undefined;' > List Title </div> <div class='listItems' style=' ; undefined; ' > item 1 </div><div class='listItems' style=' ; undefined; ' > items 2 </div><div class='listItems' style=' ; undefined; ' > item 3 </div></div>"`;

exports[`Test List List Config 3 1`] = `" <div class='listCard shadow' style=' width: fit-content;height: auto;margin: 4px 4px;; background-color: #C5CAE9; border-left: 2px solid #1565C0; undefined; ' > <div class = 'listTitle' style=' ; undefined;' > List Title </div> <div class='listItems' style=' border-bottom: 1px solid #7986CB; undefined; ' > item 1 </div><div class='listItems' style=' border-bottom: 1px solid #7986CB; undefined; ' > items 2 </div><div class='listItems' style=' border-bottom: 1px solid #7986CB; undefined; ' > item 3 </div></div>"`;

exports[`Test List List Config 4 1`] = `" <div class='listCard shadow' style=' width: fit-content;height: auto;margin: 4px 4px;; background-color: #C5CAE9; border-left: 2px solid #1565C0; undefined; ' > <div class = 'listTitle' style=' ; undefined;' > </div> <div class='listItems' style=' border-bottom: 1px solid #7986CB; undefined; ' > item 1 </div><div class='listItems' style=' border-bottom: 1px solid #7986CB; undefined; ' > items 2 </div><div class='listItems' style=' border-bottom: 1px solid #7986CB; undefined; ' > item 3 </div></div>"`;

exports[`Test Progress Bar Progress Bar Config 0 1`] = `" <div class='bar' style=' background-color:#F5F5F5 ; width: 4px; height: 100%; '> <div class='barFill' style=' background-color: #424242; width:4px; height: 75%; '> </div> </div> "`;

exports[`Test Progress Bar Progress Bar Config 1 1`] = `" <div class='bar' style=' background-color:#F5F5F5 ; width: 4px; height: 100%; '> <div class='barFill' style=' background-color: #424242; width:4px; height: 75%; '> </div> </div> "`;

exports[`Test Progress Bar Progress Bar Config 2 1`] = `" <div class='bar' style=' background-color:#F5F5F5 ; width: 100%; height: 4px; '> <div class='barFill' style=' background-color: #424242; width:75%; height: 4px; '> </div> </div> "`;

exports[`Test Progress Bar Progress Bar Config 3 1`] = `" <div class='bar' style=' background-color:#BBDEFB ; width: 100%; height: 4px; '> <div class='barFill' style=' background-color: #1565C0; width:75%; height: 4px; '> </div> </div> "`;

exports[`Test Table Table Config 0 1`] = `" <div class ='tableCard ' style='width: fit-content;height: auto;margin: 4px 4px; ; ;' > <table class='table ' style='; ;'><tbody><tr class ='tableRowOdd' style=';'><td class='tableCell' style= '; text-align: left; '> Dave </td><td class='tableCell' style= '; text-align: left; '> 10 </td><td class='tableCell' style= '; text-align: left; '> 90 </td></tr><tr class ='tableRowOdd' style=';'><td class='tableCell' style= '; text-align: left; '> Sophie </td><td class='tableCell' style= '; text-align: left; '> 9 </td><td class='tableCell' style= '; text-align: left; '> 85 </td></tr><tr class ='tableRowOdd' style=';'><td class='tableCell' style= '; text-align: left; '> Ronaldo </td><td class='tableCell' style= '; text-align: left; '> 12 </td><td class='tableCell' style= '; text-align: left; '> 45 </td></tr></tbody></table></div>"`;

exports[`Test Table Table Config 1 1`] = `" <div class ='tableCard ' style='width: fit-content;height: auto;margin: 4px 4px; ; ;' > <table class='table ' style='; ;'> <thead class='tableHead' style=' background-color: #000; ; '> <th class='tableHeadings' style='text-align: left; '> Name </th><th class='tableHeadings' style='text-align: left; '> Year </th><th class='tableHeadings' style='text-align: left; '> scores </th></thead><tbody><tr class ='tableRowOdd' style=';'><td class='tableCell' style= '; text-align: left; '> Dave </td><td class='tableCell' style= '; text-align: left; '> 10 </td><td class='tableCell' style= '; text-align: left; '> 90 </td></tr><tr class ='tableRowOdd' style=';'><td class='tableCell' style= '; text-align: left; '> Sophie </td><td class='tableCell' style= '; text-align: left; '> 9 </td><td class='tableCell' style= '; text-align: left; '> 85 </td></tr><tr class ='tableRowOdd' style=';'><td class='tableCell' style= '; text-align: left; '> Ronaldo </td><td class='tableCell' style= '; text-align: left; '> 12 </td><td class='tableCell' style= '; text-align: left; '> 45 </td></tr></tbody></table></div>"`;

exports[`Test Table Table Config 2 1`] = `" <div class ='tableCard ' style='width: fit-content;height: auto;margin: 4px 4px; ; ;' > <div class='tableTitle' style=' text-align: left; '> Student Details </div> <table class='table ' style='; ;'> <thead class='tableHead' style=' background-color: #000; ; '> <th class='tableHeadings' style='text-align: left; '> Name </th><th class='tableHeadings' style='text-align: left; '> Year </th><th class='tableHeadings' style='text-align: left; '> scores </th></thead><tbody><tr class ='tableRowOdd' style=';'><td class='tableCell' style= '; text-align: left; '> Dave </td><td class='tableCell' style= '; text-align: left; '> 10 </td><td class='tableCell' style= '; text-align: left; '> 90 </td></tr><tr class ='tableRowOdd' style=';'><td class='tableCell' style= '; text-align: left; '> Sophie </td><td class='tableCell' style= '; text-align: left; '> 9 </td><td class='tableCell' style= '; text-align: left; '> 85 </td></tr><tr class ='tableRowOdd' style=';'><td class='tableCell' style= '; text-align: left; '> Ronaldo </td><td class='tableCell' style= '; text-align: left; '> 12 </td><td class='tableCell' style= '; text-align: left; '> 45 </td></tr></tbody></table></div>"`;

exports[`Test Table Table Config 3 1`] = `" <div class ='tableCard ' style='width: fit-content;height: auto;margin: 4px 4px; border: 1px solid #2E7D32; ;' > <div class='tableTitle' style=' text-align: left; '> Student Details </div> <table class='table ' style='; ;'><tbody><tr class ='tableRowOdd' style=';'><td class='tableCell' style= '; text-align: left; '> Dave </td><td class='tableCell' style= '; text-align: left; '> 10 </td><td class='tableCell' style= '; text-align: left; '> 90 </td></tr><tr class ='tableRowOdd' style=';'><td class='tableCell' style= '; text-align: left; '> Sophie </td><td class='tableCell' style= '; text-align: left; '> 9 </td><td class='tableCell' style= '; text-align: left; '> 85 </td></tr><tr class ='tableRowOdd' style=';'><td class='tableCell' style= '; text-align: left; '> Ronaldo </td><td class='tableCell' style= '; text-align: left; '> 12 </td><td class='tableCell' style= '; text-align: left; '> 45 </td></tr></tbody></table></div>"`;

exports[`Test Table Table Config 4 1`] = `" <div class ='tableCard ' style='width: fit-content;height: auto;margin: 4px 4px; ; ;' > <div class='tableTitle' style=' text-align: left; '> Student Details </div> <table class='table ' style='; ;'> <thead class='tableHead' style=' background-color: #000; ; '> <th class='tableHeadings' style='text-align: left; '> Name </th><th class='tableHeadings' style='text-align: left; '> Year </th><th class='tableHeadings' style='text-align: left; '> scores </th></thead><tbody><tr class ='tableRowOdd' style=';'><td class='tableCell' style= '; text-align: left; '> Dave </td><td class='tableCell' style= '; text-align: left; '> 10 </td><td class='tableCell' style= '; text-align: left; '> 90 </td></tr><tr class ='tableRowOdd' style=';'><td class='tableCell' style= '; text-align: left; '> Sophie </td><td class='tableCell' style= '; text-align: left; '> 9 </td><td class='tableCell' style= '; text-align: left; '> 85 </td></tr><tr class ='tableRowOdd' style=';'><td class='tableCell' style= '; text-align: left; '> Ronaldo </td><td class='tableCell' style= '; text-align: left; '> 12 </td><td class='tableCell' style= '; text-align: left; '> 45 </td></tr></tbody></table></div>"`;

exports[`Test Table Table Config 5 1`] = `" <div class ='tableCard ' style='width: 400px;height: 400px;margin: 4px 4px; ; ;' > <div class='tableTitle' style=' text-align: left; '> Student Details </div> <table class='table ' style='; ;'> <thead class='tableHead' style=' background-color: #2E7D32; ; '> <th class='tableHeadings' style='text-align: left; '> Name </th><th class='tableHeadings' style='text-align: left; '> Year </th><th class='tableHeadings' style='text-align: left; '> scores </th></thead><tbody><tr class ='tableRowOdd' style=';'><td class='tableCell' style= '; text-align: left; '> Dave </td><td class='tableCell' style= '; text-align: left; '> 10 </td><td class='tableCell' style= '; text-align: left; '> 90 </td></tr><tr class ='tableRowOdd' style=';'><td class='tableCell' style= '; text-align: left; '> Sophie </td><td class='tableCell' style= '; text-align: left; '> 9 </td><td class='tableCell' style= '; text-align: left; '> 85 </td></tr><tr class ='tableRowOdd' style=';'><td class='tableCell' style= '; text-align: left; '> Ronaldo </td><td class='tableCell' style= '; text-align: left; '> 12 </td><td class='tableCell' style= '; text-align: left; '> 45 </td></tr></tbody></table></div>"`;

exports[`Test Table Table Config 6 1`] = `" <div class ='tableCard ' style='width: 400px;height: 400px;margin: 4px 4px; ; ;' > <div class='tableTitle' style=' text-align: left; '> Student Details </div> <table class='table ' style='; ;'> <thead class='tableHead' style=' background-color: #000; ; '> <th class='tableHeadings' style='text-align: left; '> Name </th><th class='tableHeadings' style='text-align: left; '> Year </th><th class='tableHeadings' style='text-align: left; '> scores </th></thead><tbody><tr class ='tableRowOdd' style=';'><td class='tableCell' style= '; text-align: left; '> Dave </td><td class='tableCell' style= '; text-align: left; '> 10 </td><td class='tableCell' style= '; text-align: left; '> 90 </td></tr><tr class ='tableRowOdd' style=';'><td class='tableCell' style= '; text-align: left; '> Sophie </td><td class='tableCell' style= '; text-align: left; '> 9 </td><td class='tableCell' style= '; text-align: left; '> 85 </td></tr><tr class ='tableRowOdd' style=';'><td class='tableCell' style= '; text-align: left; '> Ronaldo </td><td class='tableCell' style= '; text-align: left; '> 12 </td><td class='tableCell' style= '; text-align: left; '> 45 </td></tr></tbody></table></div>"`;

exports[`Test Table Table Config 7 1`] = `" <div class ='tableCard ' style='width: fit-content;height: auto;margin: 4px 4px; border: 1px solid #424242; ;' > <div class='tableTitle' style=' text-align: center; '> Student Details </div> <table class='table ' style='; ;'> <thead class='tableHead' style=' background-color: #000; ; '> <th class='tableHeadings' style='text-align: left; '> Name </th><th class='tableHeadings' style='text-align: left; '> Year </th><th class='tableHeadings' style='text-align: left; '> scores </th></thead><tbody><tr class ='tableRowEven' style=';'><td class='tableCell' style= '; text-align: left; '> Dave </td><td class='tableCell' style= '; text-align: left; '> 10 </td><td class='tableCell' style= '; text-align: left; '> 90 </td></tr><tr class ='tableRowOdd' style=';'><td class='tableCell' style= '; text-align: left; '> Sophie </td><td class='tableCell' style= '; text-align: left; '> 9 </td><td class='tableCell' style= '; text-align: left; '> 85 </td></tr><tr class ='tableRowEven' style=';'><td class='tableCell' style= '; text-align: left; '> Ronaldo </td><td class='tableCell' style= '; text-align: left; '> 12 </td><td class='tableCell' style= '; text-align: left; '> 45 </td></tr></tbody></table></div>"`;

exports[`Test Table Table Config 8 1`] = `" <div class ='tableCard shadow' style='width: fit-content;height: auto;margin: 4px 4px; ; ;' > <div class='tableTitle' style=' text-align: center; '> Student Details </div> <table class='table ' style='; ;'> <thead class='tableHead' style=' background-color: #283593; ; '> <th class='tableHeadings' style='text-align: left; '> Name </th><th class='tableHeadings' style='text-align: left; '> Year </th><th class='tableHeadings' style='text-align: left; '> scores </th></thead><tbody><tr class ='tableRowEven' style=';'><td class='tableCell' style= '; text-align: left; '> Dave </td><td class='tableCell' style= '; text-align: left; '> 10 </td><td class='tableCell' style= '; text-align: left; '> 90 </td></tr><tr class ='tableRowOdd' style=';'><td class='tableCell' style= '; text-align: left; '> Sophie </td><td class='tableCell' style= '; text-align: left; '> 9 </td><td class='tableCell' style= '; text-align: left; '> 85 </td></tr><tr class ='tableRowEven' style=';'><td class='tableCell' style= '; text-align: left; '> Ronaldo </td><td class='tableCell' style= '; text-align: left; '> 12 </td><td class='tableCell' style= '; text-align: left; '> 45 </td></tr></tbody></table></div>"`;

exports[`Test Table Table Config 9 1`] = `" <div class ='tableCard shadow' style='width: fit-content;height: auto;margin: 4px 4px; border: 1px solid #1565C0; ;' > <div class='tableTitle' style=' text-align: center; '> Student Details </div> <table class='table ' style='; ;'> <thead class='tableHead' style=' background-color: #000; ; '> <th class='tableHeadings' style='text-align: center; '> Name </th><th class='tableHeadings' style='text-align: center; '> Year </th><th class='tableHeadings' style='text-align: center; '> scores </th></thead><tbody><tr class ='tableRowEven' style=';'><td class='tableCell' style= '; text-align: center; '> Dave </td><td class='tableCell' style= '; text-align: center; '> 10 </td><td class='tableCell' style= '; text-align: center; '> 90 </td></tr><tr class ='tableRowOdd' style=';'><td class='tableCell' style= '; text-align: center; '> Sophie </td><td class='tableCell' style= '; text-align: center; '> 9 </td><td class='tableCell' style= '; text-align: center; '> 85 </td></tr><tr class ='tableRowEven' style=';'><td class='tableCell' style= '; text-align: center; '> Ronaldo </td><td class='tableCell' style= '; text-align: center; '> 12 </td><td class='tableCell' style= '; text-align: center; '> 45 </td></tr></tbody></table></div>"`;

exports[`Test Table Table Config 10 1`] = `" <div class ='tableCard ' style='width: fit-content;height: auto;margin: 4px 4px; ; ;' > <div class='tableTitle' style=' text-align: center; '> Student Details </div> <table class='table ' style=' border: 1px solid #1565C0; ;'> <thead class='tableHead' style=' background-color: #000; ; '> <th class='tableHeadings' style='text-align: center; '> Name </th><th class='tableHeadings' style='text-align: center; '> Year </th><th class='tableHeadings' style='text-align: center; '> scores </th></thead><tbody><tr class ='tableRowEven' style=';'><td class='tableCell' style= '; text-align: center; '> Dave </td><td class='tableCell' style= '; text-align: center; '> 10 </td><td class='tableCell' style= '; text-align: center; '> 90 </td></tr><tr class ='tableRowOdd' style=';'><td class='tableCell' style= '; text-align: center; '> Sophie </td><td class='tableCell' style= '; text-align: center; '> 9 </td><td class='tableCell' style= '; text-align: center; '> 85 </td></tr><tr class ='tableRowEven' style=';'><td class='tableCell' style= '; text-align: center; '> Ronaldo </td><td class='tableCell' style= '; text-align: center; '> 12 </td><td class='tableCell' style= '; text-align: center; '> 45 </td></tr></tbody></table></div>"`;

exports[`Test Table Table Config 11 1`] = `" <div class ='tableCard ' style='width: fit-content;height: auto;margin: 4px 4px; ; ;' > <div class='tableTitle' style=' text-align: center; '> Student Details </div> <table class='table shadow' style='; ;'> <thead class='tableHead' style=' background-color: #000; ; '> <th class='tableHeadings' style='text-align: center; '> Name </th><th class='tableHeadings' style='text-align: center; '> Year </th><th class='tableHeadings' style='text-align: center; '> scores </th></thead><tbody><tr class ='tableRowEven' style=';'><td class='tableCell' style= '; text-align: center; '> Dave </td><td class='tableCell' style= '; text-align: center; '> 10 </td><td class='tableCell' style= '; text-align: center; '> 90 </td></tr><tr class ='tableRowOdd' style=';'><td class='tableCell' style= '; text-align: center; '> Sophie </td><td class='tableCell' style= '; text-align: center; '> 9 </td><td class='tableCell' style= '; text-align: center; '> 85 </td></tr><tr class ='tableRowEven' style=';'><td class='tableCell' style= '; text-align: center; '> Ronaldo </td><td class='tableCell' style= '; text-align: center; '> 12 </td><td class='tableCell' style= '; text-align: center; '> 45 </td></tr></tbody></table></div>"`;

exports[`Test Text Text Config 0 1`] = `"<div class='text' style= 'font-size: 40px; color: #1565C0; ; '> This is test text </div>"`;

exports[`Test Text Text Config 1 1`] = `"<div class='text' style= 'font-size: 30px; color: #2E7D32; ; '> This is test text </div>"`;

exports[`Test Text Text Config 2 1`] = `"<div class='text' style= 'font-size: 20px; color: #000; ; '> This is test text </div>"`;

exports[`Test Text Text Config 3 1`] = `"<div class='text' style= '; color: #000; ; '> This is test text </div>"`;

exports[`Test Text Text Config 4 1`] = `"<div class='text' style= 'font-size: 15px; color: #000; ; '> This is test text </div>"`;

exports[`Test Text Text Config 5 1`] = `"<sub class='subText' style= 'color: #000'> This is test text ; </sub>"`;
